packages/graphql/src/translate/queryAST/ast/fields/aggregation-fields/CountField.ts

```diff
@@ -51,7 +51,7 @@ export class CountField extends AggregationField {
     }
 
     public getAggregationExpr(variable: Cypher.Variable): Cypher.Expr {
-        return Cypher.count(variable);
+        return Cypher.count(variable).distinct();
```

Comment:
Can you add this in a separate PR?
```suggestion
        return Cypher.count(variable).distinct();
```